css/style.css

```diff
@@ -16,4 +15,210 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* Header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px 75px;
+}
+
+.karma-icon {
+  width: 50px;
+  height: 50px;
+}
+
+.menu-hamburger {
+  display: none; /* Only for mobile */
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-list {
+  list-style: none;
+  margin-right: 50px;
+}
+
+.nav-link {
+  color: #adadad;
+  text-decoration: none;
+}
+
+.meet-karma {
+  color: #000;
+  font-weight: 500; /* This for the Meet Karma button */
+}
+
+.nav-link:hover,
+.meet-karma:hover {
+  color: #f15b2a;
+}
+
+/* Main */
+.main {
+  margin-bottom: 5%;
+}
 
+/* Intro section */
+.intro {
+  background-image: url("../img/first-background.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 735px;
+  color: #fff;
+  text-align: center;
+  margin-bottom: 5%;
+}
+
+.intro-title {
+  padding-top: 10%;
+  font-weight: 300;
+  font-size: 50px;
+  margin-bottom: -15px;
+}
+
+.description {
+  font-size: 40px;
+  font-weight: 100;
+  margin-bottom: 55px;
+}
+
+.learn-more-button {
+  background-color: #f15b2a;
+  border-radius: 2px;
+  border: none;
+  padding: 15px 35px;
+}
+
+/* Info section */
+.info {
+  text-align: center;
+  margin-bottom: 7%;
+}
+
+.info-text {
+  font-weight: 300;
+  font-size: 40px;
+  margin-bottom: 50px;
+}
+
+.services {
+  display: flex;
+  justify-content: space-around; /* Flexbox to show services */
+}
+
+.service-icon {
+  width: 100px;
+  height: 100px;
+}
+
+.service-text {
+  font-size: 25px;
+}
+
+.quote {
+  display: flex; /* Used flexbox so that image and quote can go side by side */
+}
+
+.quote-image-box {
+  background-image: url("../img/feature-img.png");
+  background-repeat: no-repeat;
+  padding: 11% 15%;
+}
+
+.quote-text-box {
+  background-color: #fff2f2;
+  width: 100%;
+}
+
+.quote-text {
+  font-size: 35px;
+  margin: 15% 12% 25px;
+}
+
+.quote-orange {
+  color: #f15b2a;
+  font-weight: 600;
+}
+
+.quote-button {
+  margin: 0 40% 10%;
```

Comment:
This margin is forcing the button to be too small and wrap onto several lines. 
You should need any left and right margin - just size the button using padding, and center it using flexbox.